native/src/hooks/useLoadExtraCityContent.ts

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from 'react'
```

Comment:
As the comment says this is just for easier handling if city content (i.e. offline available content) and something else needs to be loaded at the same time. Otherwise both responses (data, loading, error,  refresh) would need to be checked. Used for disclaimer, offers, sprungbrett offers.